### PR TITLE
feat: Create processor binary release

### PR DESCRIPTION
**Pull request by toninis**
#### Summary

Created specific release for processor binaries
Related discussion [here](https://hub.mattermost.com/private-core/pl/jyeiok9zo38a5dui6468br8syc)

#### Ticket Link

https://mattermost.atlassian.net/browse/CLD-8042

### CODE REVIEW

**Comment by toninis** [comment]
@wiggin77 I incorporated the changes from https://github.com/mattermost/mattermost-plugin-legal-hold/pull/95 here . 
We can close it 🚀

File: .github/workflows/processor-release.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+      - name: Setup Node
+        uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b # v4.0.3
+        with:
+          node-version-file: .nvmrc
```
Comment (saturninoabril) [inline .github/workflows/processor-release.yml:18]: Maybe node is not needed for processor build?

**Review by saturninoabril — APPROVED** [review]
Awesome, thanks!

**Review by wiggin77 — APPROVED** [review]
LGTM 👍

**Pull request merged**